src/HSE/Util.hs

```diff
@@ -296,6 +297,8 @@ an = toSrcInfo nullSrcLoc [] nullSrcLoc
 dropAnn :: Functor f => f s -> f ()
 dropAnn = void
 
+instance Default SrcSpanInfo where
+  def = an
```

Comment:
Any reason this instance shouldn't live in haskell-src-exts-util?

Reply:
Also, where is the `def` value used? Does one of the HSE-util things have it as required context?

Reply:
It is used by the rebracketing code to insert `Paren` constructors. 

You are probably right that the instance could live in HSE-util. I never know with orphan instances, but in this case there is only one sensible default value and it would make HSE-util immediately easier to use.

Reply:
Yep, I think that's the thing that should be pushed up to haskell-src-exts first - it's obvious useful. Having it in HSE-util just makes the transition easier.